R/infillOptCMAES.R

```diff
@@ -44,12 +44,23 @@ infillOptCMAES = function(infill.crit, models, control, par.set, opt.path, desig
     )))
   }
 
+  if (!("restart.triggers " %in% names(names(cmaes.control)))) {
+    cmaes.control = BBmisc::insert(cmaes.control, list(
+      restart.triggers = BBmisc::extractSubList(getDefaultStoppingConditions(), "name")
+    ))
+  }
+
   # set number of restarts
   cmaes.control = BBmisc::insert(cmaes.control, list(max.restarts = control$infill.opt.restarts))
 
   # select first start point as currently best
-  start.point = unlist(getOptPathEl(opt.path, getOptPathBestIndex(opt.path))$x)
-  result = cmaesr::cmaes(fn, start.point = start.point, monitor = NULL, control = cmaes.control)
+  # FIXME: Intelligent start point for multi crit
+  if (control$n.objectives == 1L) {
+    start.point = unlist(getOptPathEl(opt.path, getOptPathBestIndex(opt.path))$x)
```

Comment:
Is this even a smart idea? Aren't we more likely to run into a local minimum close to the old optimum? Does this effectively kill exploration, because a global maxima because of a high se value is unlikely to be found?

Reply:
Again: Old Code from Jakob B. I did just disable it for the multi-objective case.

And I agree with you.